### ISSUE TITLE
README memory benchmark results are outdated (in a bad sense)

### ISSUE THREAD

**Issue by funny-falcon-at-joomcode** [mediocregopher/radix.v3#42]
```
~/go/src/github.com/mediocregopher/radix.v3$ go test -run FoO -bench GetSet -benchmem
goos: linux
goarch: amd64
pkg: github.com/mediocregopher/radix.v3
BenchmarkSerialGetSet/radix-8              50000             36969 ns/op            1092 B/op         18 allocs/op
BenchmarkSerialGetSet/redigo-8             50000             34345 ns/op              86 B/op          5 allocs/op
BenchmarkParallelGetSet/radix-8           100000             12359 ns/op            1126 B/op         19 allocs/op
BenchmarkParallelGetSet/redigo-8          200000             11311 ns/op             121 B/op          6 allocs/op
PASS
ok      github.com/mediocregopher/radix.v3      8.285s
```

**Comment by mediocregopher** [comment]
Hi @funny-falcon-at-joomcode, thanks for bringing this up to me! Your benchmarks are being run on an amd64 machine which is why you're seeing different benchmark results than I have printed in the README (which are for x86_64). I am able to reproduce your results on my chromebook (which is an amd64 machine), but I haven't had the chance to look into where the disparity is coming from. I'll update this when I do, hopefully in the next week or two.

**Comment by funny-falcon-at-joomcode** [comment]
> amd64 machine which is why you're seeing different benchmark results than I have printed in the README (which are for x86_64).

amd64 IS EXACTLY SAME TO x86_64: https://en.wikipedia.org/wiki/X86-64
> x86-64 (also known as x64, x86_64, AMD64 and Intel 64[note 1])
> note 1: Various names are used for the instruction set. Prior to the launch, x86-64 and x86_64 were used, while upon the release AMD named it AMD64.[1] Intel initially used the names IA-32e and EM64T before finally settling on "Intel 64" for its implementation. Some in the industry, including Apple,[2][3][4] use x86-64 and x86_64, while others, notably Sun Microsystems[5] (now Oracle Corporation) and Microsoft,[6] use x64. The BSD family of OSs and several Linux distributions[7][8] use AMD64, as does Microsoft Windows internally.[9][10]

**Comment by nussjustin** [comment]
Opened #43, which should improve allocations again.

**Comment by mediocregopher** [comment]
Ack my bad, I got "amd64" confused with "arm64", which is what I have with
me. I'll take a look at Justin's fix in the morning.

**Comment by mediocregopher** [comment]
merged #43, i'll leave this open till I get home on friday and double check on my desktop that x86_64/amd64/whatever looks right

**Comment by mediocregopher** [comment]
@funny-falcon-at-joomcode after justin's change the allocations looks back to normal on my computer, if you wanna double check them on yours you can re-open this if they still don't look right. Thanks for bringing this up!
